### PR TITLE
streams are already set non-blocking in Stream::__construct()

### PR DESCRIPTION
**Pull request by staabm**
see https://github.com/reactphp/stream/blob/master/src/Stream.php#L26

### CODE REVIEW

**Comment by clue** [comment]
Please note that this has only been added with [react/stream:v0.4.2](https://github.com/reactphp/stream/blob/master/CHANGELOG.md#042-2014-09-09).

Thus, if we want to rely on this, we have to update our minimum version requirement.

**Comment by staabm** [comment]
@clue thanks for the hint. adjusted the composer.json accordingly

File: composer.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
         "php": ">=5.4.0",
         "evenement/evenement": "~2.0",
         "react/event-loop": "0.4.*",
-        "react/stream": "0.4.*"
+        "react/stream": ">=0.4.2"
```
Comment (jsor) [inline composer.json]: This should be `~0.4.2` if we want to stay in `0.4` branch.

**Comment by staabm** [comment]
@jsor fixed

**Comment by cboden** [comment]
LGTM

**Comment by WyriHaximus** [comment]
LGTM :+1:

**Comment by clue** [comment]
LGTM :+1:

**Comment by staabm** [comment]
@jmikola ping

**Comment by staabm** [comment]
@jmikola ping

**Pull request merged**